app/workers/crawler/my_basket.rb

```diff
@@ -15,14 +15,18 @@ def fetch_places
     shop_names = page.xpath('//tr[@class="vcard"]/td[@class="fn org"]').map{|node| node.text }
     postal_codes = page.xpath('//tr[@class="vcard"]/td[@class="postal-code"]').map{|node| node.text }
     prefectures = page.xpath('//tr[@class="vcard"]/td[@class="adr"]').map { |node| node.xpath('./span[@class="region"]').text.squish.match(/東京都|神奈川県/).to_s }
-    addresses = page.xpath('//tr[@class="vcard"]/td[@class="adr"]').map { |node| node.xpath('./span[@class="locality"]').text + node.xpath('./span[@class="street-address"]').text }
+    addresses = page.xpath('//tr[@class="vcard"]/td[@class="adr"]').map { |node| node.xpath('./span[@class="locality"]').text + node.xpath('./span[@class="street-address"]').text.squish }
     telephones = page.xpath('//tr[@class="vcard"]/td[@class="tel"]').map{|node| node.text }
     shop_names.each_with_index do|shop, index|
-      unless prefectures[index].empty?
+      temp_address = addresses[index].gsub('　', ' ').split(' ')
```

Comment:
__[rubocop]__
* Use `tr` instead of `gsub`. [:ref](https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)